.svgrrc.json

```diff
@@ -1,5 +1,4 @@
 {
-  "svgo": false,
```

Comment:
I didn't necessarily to fix the bug, but it should help optimise the generated `.svg` files
We had it all configured (`svgo.config.js`) but disabled here, can't remember why
Though the output is still pretty much all the same, so I don't see any downsides in re-enabling it

Reply:
I'll split the commits for clarity

Reply:
So yet again - the SVGO is run, but not _during_ the build process but _during_ commit.

So when a new SVG is added - during commit it is _then_ optimized.

To avoid double optimization (and TBH I don't even know if this one picks up our `svgo.config.js`) this one was disabled.

Downside of this being a JSON that a comment explaining could not be left...

Reply:
What about Boxicons, are they ever optimised?

Reply:
I believe they are already - you can try to build with and without the setting, and see if there's a difference.

Also, if it _does_ pickup the configuration I don't see a problem optimizing twice 😏